packages/apidom-ls/src/config/openapi/header/lint/schema--mutually-exclusive-openapi-3-1-asyncapi-2.ts

```diff
@@ -0,0 +1,26 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
```

Comment:
Generally I failed to understand this rule as we had already https://github.com/swagger-api/apidom/pull/3640/files#diff-fae1e7019c75a06eac6d526782e8ab00d2ee2ce0f48391261d629686bc995c9a working for OpenAPI 3.x.y.